Makefile

```diff
@@ -0,0 +1,126 @@
+.DEFAULT_GOAL	:= build
+
+#------------------------------------------------------------------------------
+# Variables
+#------------------------------------------------------------------------------
+
+SHELL 		:= /bin/bash
+BINDIR		:= bin
+BUILDDIR    := build
+DOCKERDIR	:= docker
+RELEASEDIR  := release
+OUTPUT_NAME := envoyctl
```

Comment:
should this be `go-control-plane`?

Reply:
envoyctl is so cool! :( .. Either way, I thought that this would change rapidly, when we start putting in CLIs in cmd/ folder.. We need one per CLI tool.